engine/src/main/java/org/terasology/rendering/nui/widgets/UIDropdown.java

```diff
@@ -137,7 +137,7 @@ public Vector2i getPreferredContentSize(Canvas canvas, Vector2i areaHint) {
     public String getMode() {
         if (!isEnabled()) {
             return DISABLED_MODE;
-        } else if (opened || (TabbingManager.focusedWidget != null && TabbingManager.focusedWidget.equals(this))) {
```

Comment:
The removed bit is important! It needs to know if the dropdown is the focusedWidget (in other words, if it is the current selection by tabbing) in order to highlight the widget with yellow - that's what returning `ACTIVE_MODE` does. You can probably find another workaround to avoid using `TabbingManager`, but you need some way to tell if it's been tabbed to 🙂

Reply:
Got it! Btw, I've spotted, that TabbingManager resets every time when mouse clicking or mousewheel turns (at processMouseClick method of nui\internal\CanvasImpl class). Not an issue, just the matter of opinion. Do you think, that it is more comfortable to start over every time widget gets inactive or more convinient will be highlight the next one after tabbing?

Reply:
Without looking over that code section again (so I might be mistaken), I would think that it is meant to keep it sort of consistent - if you click somewhere else, tabbing shouldn't continue from that random spot that it was at before the click. I guess the way to deal with it at the moment is using the method that you mentioned, so that the player knows that tabbing will start from the beginning of the widgets. If that plan were to change, it should end up starting tabbing from the freshly clicked widget.